package.json

```diff
@@ -3,25 +3,31 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "axios": "^0.18.0",
     "enzyme": "^3.9.0",
     "enzyme-adapter-react-16": "^1.10.0",
     "jest-enzyme": "^7.0.1",
+    "jest-fetch-mock": "^2.1.1",
+    "prop-types": "^15.7.2",
     "react": "^16.8.3",
     "react-dom": "^16.8.3",
+    "react-facebook-login": "^4.1.1",
+    "react-google-login": "^5.0.2",
+    "react-redux": "^6.0.1",
     "react-router-dom": "^4.3.1",
     "react-scripts": "2.1.5",
     "react-test-renderer": "^16.8.3",
-    "redux-mock-store": "^1.5.3",
-    "jest-fetch-mock": "^2.1.1",
-    "react-redux": "^6.0.1",
     "redux": "^4.0.1",
+    "redux-mock-store": "^1.5.3",
     "redux-thunk": "^2.3.0",
-    "redux-thunk-actions": "^1.3.2"
+    "redux-thunk-actions": "^1.3.2",
+    "semantic-ui-css": "^2.4.1",
+    "semantic-ui-react": "^0.85.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "HTTPS=true react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
+    "test": "react-scripts test -u",
```

Comment:
Good job on this feature @missvicki  thanks for the one on one explanation. though it might not be a good idea to leave the `-u` flag on your test script. It will keep replacing your snapshots with new ones, hence eroding the whole point of snapshot testing.

Reply:
Thank you for spotting this, I will make the correction.